Adding Azure tenancy for preview (#5877)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/secure/snowflake-privatelink.md

<SetUpPages features={'/snippets/_available-tiers-privatelink.md'}/>

> The following steps will walk you through the setup of a Snowflake AWS PrivateLink or Azure Private Link endpoint in the dbt Cloud multi-tenant environment.

Reviewer: ```suggestion
The following steps walk you through the setup of a Snowflake AWS PrivateLink and Azure Private Link endpoint in the dbt Cloud multi-tenant environment.
```
